src/components/common/Page.tsx

```diff
@@ -0,0 +1,20 @@
+import { FunctionComponent, ReactNode } from 'react';
```

Comment:
Should be renamed to something that gives user a better understanding of what the component is doing.